src/v2/api/index.md

```diff
@@ -509,6 +509,17 @@ type: api
 
   A list/hash of attributes that are exposed to accept data from the parent component. It has an Array-based simple syntax and an alternative Object-based syntax that allows advanced configurations such as type checking, custom validation and default values.
 
+  With Object-based syntax, you can use following options:
+    - **type:** can be one of the following native constructors: `String`, `Number`, `Boolean`, `Array`, `Object`, `Date`, `Function`, `Symbol` or any custom constructor function. Will check if a prop has a given type, and will throw a warning if it doesn't.
```

Comment:
It can also be an array of those, like
```javascript
type: [String, Number]
```

Consider adding a `"More"` link to: https://vuejs.org/v2/guide/components-props.html#Prop-Types

Reply:
Fixed! 👍